tests/mock/conftest.py

```diff
@@ -10,8 +10,20 @@
 
 
 @pytest.fixture(scope="function")
-def session():
-    with Session(MockServerConnection()) as s:
+def mock_server_connection(multithreading_mode_enabled):
+    options = {
```

Comment:
local testing does not make real connection so testing enabled/disabled mode using parameter.